README.md

```diff
@@ -53,6 +53,7 @@ build tool to build the binary.
 git clone https://github.com/hadolint/hadolint
 cd hadolint
 stack build
```

Comment:
Thanks! Good idea @samiralajmovic. 

In this case, we can probably drop `stack build`.